src/Core/Prefab/GlobeView.js

```diff
@@ -371,10 +371,17 @@ GlobeView.prototype.readDepthBuffer = function readDepthBuffer(x, y, width, heig
 
 const matrix = new THREE.Matrix4();
 const screen = new THREE.Vector2();
-const pickWorldPosition = new THREE.Vector3();
 const ray = new THREE.Ray();
 const direction = new THREE.Vector3();
-GlobeView.prototype.getPickingPositionFromDepth = function getPickingPositionFromDepth(mouse) {
+
+/**
+ * returns the pick position under the mouse
```

Comment:
Sorry, but this isn't a documentation.

What is a "pick position"?
When should I use this function?
Why does the function name ends with `FromDepth`?

If you don't answer this kind of questions, the documentation is useless.